packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -363,9 +373,30 @@ export default class DatePicker extends React.Component {
       this.setState({
         preSelection: date
       })
+
+      let ariaLiveMessage = '';
+      if (date) {
+        const localizedDate = localizeDate(date, this.props.locale);
+        ariaLiveMessage = localizedDate.format('LL');
+
+        if (isDayDisabled(date, this.props)) {
+          ariaLiveMessage = ariaLiveMessage.concat(' ', this.props.intl.formatMessage({ id: 'Terra.datePicker.disabled' }));
+        } 
+      }
+
+      this.updateAriaLiveStatus(ariaLiveMessage);
     }
   }
 
+  updateAriaLiveStatus(message) {
+    this.ariaLiveStatus = message;
+
+    // Clears status so aria live announces correctly next time a date is preselected.
+    setTimeout(() => {
+      this.ariaLiveStatus = '';
```

Comment:
How did we arrive at 1 second for this?

Reply:
I'm using the same timeout as [select](https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/_Menu.jsx#L253) and [infinite list](https://github.com/cerner/terra-framework/blob/master/packages/terra-infinite-list/src/InfiniteList.jsx#L166). 1 second is enough to allow enough time for the re-render and announce the text. Half a second might be a little too quick to reset it.